misc/dist/linux/godot.6

```diff
@@ -1,4 +1,4 @@
-.TH GODOT "6" "February 2018" "godot 3.1" "Games"
+.TH GODOT "6" "Februaryot 2018" "godot 3.1" "Games"
```

Comment:
Newspeak for July? ;-)

Reply:
Whelp, I'm not sure how that happened :D